elasticsearch/probe/readiness.sh

```diff
@@ -21,7 +21,7 @@ export NSS_SDB_USE_CACHE=no
 # TODO: try re-use code from ./run.sh
 ES_REST_BASEURL=https://localhost:9200
 EXPECTED_RESPONSE_CODE=200
-secret_dir=/etc/elasticsearch/secret
```

Comment:
if we change this then the configmap will be incorrect https://github.com/openshift/elasticsearch-operator/blob/feature-es6x/pkg/k8shandler/configuration_tmpl.go#L38